eng/common/internal/Tools.csproj

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
```

Comment:
The readme in eng/common says to not modify files here.

https://github.com/dotnet/roslyn/blob/e959d2a9c9b27cde09430f7755ff48bc2825d66e/eng/common/README.md

Reply:
Good catch! I will revert.